config/crd/bases/awx.ansible.com_awxs.yaml

```diff
@@ -57,6 +57,10 @@ spec:
               hostname:  # deprecated
                 description: (Deprecated) The hostname of the instance
                 type: string
+              create_superuser:
```

Comment:
Since the admin_user and admin_password_secret parameters have already been part of the operator for a while, could we be consistent with the naming of this param and use `create_admin_user` as the param name?